Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -35,13 +35,14 @@ module SharedEnvExtension
 
   sig {
     params(
-      formula:      T.nilable(Formula),
-      cc:           T.nilable(String),
-      build_bottle: T.nilable(T::Boolean),
-      bottle_arch:  T.nilable(T::Boolean),
+      formula:         T.nilable(Formula),
+      cc:              T.nilable(String),
+      build_bottle:    T::Boolean,
+      bottle_arch:     T.nilable(String),
+      testing_formula: T::Boolean,
```

Comment:
I don't understand what this argument is needed for, it doesn't seem to be used anywhere. Regardless: a testing-only parameter feels a little strange/brittle here?

Reply:
It was added in af6be13e7c8e07ee1d8374d587c1e03184d0ee08, but was never applied to the generic OS case.

Reply:
Looking at that, I could probably simplify some of the overrides back to `**options` again.

Reply:
Ah, gotcha. I assumed this was something to do with `rspec` tests rather than `brew test` (which seems to be fine to be a bit different). Ignore me!